runtime/syntax/java.vim

```diff
@@ -411,6 +385,10 @@ if exists("java_highlight_functions")
   syn match javaLambdaDef "\k\@4<!(\%(\k\|[[:space:]<>?\[\]@,.]\)*)\s*->"
 endif
 
+" The @javaTop cluster comprises non-contained Java syntax groups.
```

Comment:
The @javaTop cluster was removed, so this comment needs to be updated.

Reply:
No, `@javaTop` is still defined.  Look below.

Reply:
ah yes obviously. Sorry.